scripts/sign_rproc_fw.py

```diff
@@ -0,0 +1,576 @@
+""")
+    raise
+
+try:
+    from Cryptodome.Hash import SHA256
+    from Cryptodome.Signature import pkcs1_15
+    from Cryptodome.PublicKey import RSA
+    from Cryptodome.Signature import DSS
+    from Cryptodome.PublicKey import ECC
+except ImportError:
+    print("""
+***
+ERROR: "pycryptodomex" python module should be installed.
+***
+""")
+    raise
+
+import os
+import sys
+import struct
+import logging
+import binascii
+
+#  Generated file structure:
+#
+#                    ----+-------------+
+#                   /    |    Magic    |   32-bit word, magic value equal to
+#                  /     +-------------+   0x3543A468
+#                 /      +-------------+
+#                /       |   version   |   32-bit word, version of the format
+#               /        +-------------+
+#              /         +-------------+
+# +-----------+          |  sign size  |   32-bit word, size of the signature
+# |   Header  |          +-------------+   (in bytes, 64-bit aligned)
+# +-----------+          +-------------+
+#              \         | images size |   32 bit word, size of the images to
+#               \        +-------------+   load (in bytes, 64-bit aligned)
+#                \       +-------------+
+#                 \      |  TLV size   |   32 bit word, generic TLV size
+#                  \     +-------------+   (in bytes, 64-bit aligned)
+#                   \    +-------------+
+#                    \   | PLAT TLV sz |   32 bit word, platform TLV size
+#                     \--+-------------+   (in bytes, 64-bit aligned)
+#                        +-------------+
+#                        |     TLV     |   Information used to authenticate the
+#                        |             |   firmware, stored in
+#                        +-------------+   Type-Length-Value format.
+#                        +-------------+
+#                        |   Platform  |   Specific platform information,
+#                        |     TLV     |   stored in Type-Length-Value format.
+#                        +-------------+
+#                        +-------------+   Signature of the header, the TLV,
+#                        | Signature   |   and the platform TLV.
+#                        +-------------+
+#
+#                        +-------------+
+#                        |   Firmware  |
+#                        |    image 1  |
+#                        +-------------+
+#                               ...
+#                        +-------------+
+#                        |   Firmware  |
+#                        |    image n  |
+#                        +-------------+
+#
+#  TLV and platform TLV chunk structure:
+#
+#                  ------+-------------+
+#                  |     |  TLV Magic  |  32-bit word, magic value 0x69070000
+#     +------------+     +-------------+  (TLV) or 0x69080000 (platform TLV)
+#     | TLV header |     +-------------+
+#     +------------+     |  TLV size   |  32-bit word, size of the TLV payload
```

Comment:
"TLV size" is already defined in the main header, no need to repeat it here. So the TLV header isn't needed.